app/controllers/recent_contribution_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# Controller to fetch most recent contribution
+class RecentContributionController < ApplicationController
```

Comment:
I think this might should be called ItemsController, and we should add a parameter which allows you to filter by Seller — just to follow general REST standards

Reply:
If we go with this route, we'll put ourselves in a position where we'll have a new endpoint for every feature. I'm no expert at API design, but I usually reference Stripe's API as an example when designing APIs

Reply:
Good point! I’d guess having a generic items endpoint which takes in the query you’d like would be most optimal? Such as /sellers/items?sort=desc&limit=1 ?

Reply:
Yeah totally